spec/models/mixins/supports_feature_mixin_spec.rb

```diff
@@ -200,4 +205,56 @@ def initialize(options = {})
       expect(NukeablePost.new(:bribe => false).supports_nuke?).to be true
     end
   end
+
+  describe ".subclasses_supporting" do
```

Comment:
Need to a spec for `.supporting`

Reply:
getting our caching of descendants to not blow up between tests was the hardest part of this code.
But I'm pretty sure we got it working.